airbyte-integrations/connectors/source-bigquery/unit_tests/integration/utils.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import tempfile
+import traceback
+from io import StringIO
+from pathlib import Path
+from typing import Any, List, Mapping, Optional, Union
+
+from airbyte_cdk.entrypoint import AirbyteEntrypoint
+from airbyte_cdk.exception_handler import assemble_uncaught_exception
+from airbyte_cdk.logger import AirbyteLogFormatter
+from airbyte_cdk.sources import Source
+from airbyte_protocol.models import (
+    AirbyteLogMessage,
+    AirbyteMessage,
+    AirbyteStateMessage,
+    AirbyteStreamStatus,
+    ConfiguredAirbyteCatalog,
+    Level,
+    TraceType,
+    Type,
+)
+from pydantic.error_wrappers import ValidationError
+from airbyte_cdk.test.entrypoint_wrapper import EntrypointOutput, make_file
+
+
+def discover(
```

Comment:
Thanks for defining this. We added this as part of airbyte-cdk:1.3.3 if you want to rely on it. See the changes [here](https://github.com/airbytehq/airbyte/pull/39396)

Reply:
Curretnly I prefer not to, as I already tried and had breaking changes in the CDK.